Makefile

```diff
@@ -122,12 +117,11 @@ help: ## Show this help. Example: make help
         check-release \
         coverage \
         dist \
-        format \
```

Comment:
I've typically seen it where the `.PHONY` statements are above each command so that you don't run into a case where one is added or removed and the `.PHONY` command is not. Would you be against me making that change, @zstumgoren?